addons/l10n_hk/data/template/account.account-hk.csv

```diff
@@ -57,7 +57,7 @@
 "l10n_hk_5214","Insurance","5214","expense","account.account_tag_operating","False"
 "l10n_hk_5215","Sales Commission","5215","expense","account.account_tag_operating","False"
 "l10n_hk_5216","Overseas Traveling","5216","expense","account.account_tag_operating","False"
-"l10n_hk_5217","MPF Contribution","5217","expense","account.account_tag_operating","False"
+"l10n_hk_5217","MPF (Employer) Contribution","5217","expense","account.account_tag_operating","False"
```

Comment:
```suggestion
"l10n_hk_5217","MPF Contribution (Employer)","5217","expense","account.account_tag_operating","False"
```

Wouldn't it be clearer? Otherwise LGTM